content/rosa/rosa-hcp-portworx/index.md

```diff
@@ -0,0 +1,249 @@
+---
+date: '2024-04-20'
+title: Install Portworx on Red Hat OpenShift Service on AWS (ROSA) with hosted control planes (HCP)
+tags: ["ROSA-HCP", "Portworx", "ROSA"]
+authors:
+  - Nerav Doshi
+---
+
+Portworx storage is a built-for-Kubernetes service that offers flexible and scalable persistent storage for applications in production. In this tutorial we will look at installing Portworx Enterprise on ROSA-HCP. 
+
+## Prerequisites
+You must have a Red Hat OpenShift Service on AWS (ROSA) with [hosted control plane cluster](https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-sts-overview-of-the-default-cluster-specifications_rosa-hcp-sts-creating-a-cluster-quickly) 
+Your cluster must be running OpenShift 4 or higher
+Your cluster must meet AWS prerequisites for ROSA-HCP
+* [Create Portworx user and set policies](https://docs.portworx.com/portworx-enterprise/install-portworx/openshift/rosa/aws-redhat-openshift#create-a-portworx-user)
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+* [Rosa CLI](https://github.com/openshift/rosa/releases/tag/v1.2.37) v1.2.37
+
+## Generate Portworx spec
+
+1. Navigate to [Portworx Central](https://central.portworx.com/landing/login) and log in, or create an account.
+
+2. Click Get Started. Select Portworx Enterprise from the Product Catalog page.
+
+3. On the Product Line page, choose any option depending on which license you intend to use, then click Continue to start the spec generator.
+
+4. For Platform, choose **AWS**. Under Distribution Name, select **Red Hat OpenShift Service on AWS (ROSA)**. Enter your cluster's Kubernetes version, then click **Customize** at the bottom of the window.
+
+3. Click Next and navigate to the Customize window to specify your previously created AWS security credentials in the Environment Variables section as follows:
+
+    - name: AWS_ACCESS_KEY_ID; value: <your-aws-access-key>
```

Comment:
is there a better way to do this than past AWS creds into a portworx website ?

Reply:
Incorporated adding secrets as environment variables